src/commands/deleteManagedEnvironment/DeleteManagedEnvironmentStep.ts

```diff
@@ -3,24 +3,24 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { ContainerAppsAPIClient } from "@azure/arm-appcontainers";
+import type { ContainerAppsAPIClient } from "@azure/arm-appcontainers";
 import { AzureWizardExecuteStep, parseError } from "@microsoft/vscode-azext-utils";
-import { Progress, window } from "vscode";
+import type { Progress } from "vscode";
 import { ext } from "../../extensionVariables";
 import { createContainerAppsAPIClient } from "../../utils/azureClients";
 import { localize } from "../../utils/localize";
-import { IDeleteManagedEnvironmentWizardContext } from "./IDeleteManagedEnvironmentWizardContext";
+import type { IDeleteManagedEnvironmentContext } from "./IDeleteManagedEnvironmentContext";
 
-export class DeleteManagedEnvironmentStep extends AzureWizardExecuteStep<IDeleteManagedEnvironmentWizardContext> {
+export class DeleteManagedEnvironmentStep extends AzureWizardExecuteStep<IDeleteManagedEnvironmentContext> {
     public priority: number = 110;
 
-    public async execute(context: IDeleteManagedEnvironmentWizardContext, progress: Progress<{ message?: string | undefined; increment?: number | undefined }>): Promise<void> {
-        const deleting: string = localize('deletingManagedEnv', 'Deleting Container Apps environment "{0}"... this may take several minutes.', context.managedEnvironmentName);
+    public async execute(context: IDeleteManagedEnvironmentContext, progress: Progress<{ message?: string | undefined; increment?: number | undefined }>): Promise<void> {
         const client: ContainerAppsAPIClient = await createContainerAppsAPIClient([context, context.subscription]);
 
+        const deleting: string = localize('deletingManagedEnv', 'Deleting environment... this may take several minutes...');
```

Comment:
I prefer having the name in here since it takes so long. Also don't like the added ellipsis at the end.

Reply:
What, really? It's already in the title though

Reply:
I tried without ellipses, and it looks kind of weird.  What phrase are you looking at in total as the replacement?

Reply:
![image](https://github.com/microsoft/vscode-azurecontainerapps/assets/40250218/8b2d633a-06bf-4616-a8ba-26f742b0ef4a)

Reply:
Ahhh sorry, I forgot that's how it looked on the activity log.

Maybe we just chnage it to `This may take a few minutes...`?

Reply:
I'm also cool with that, but let me give you some additional context just in case.  

The reason I kept the `Deleting environment...` portion prefixed was because the description could first show the `Deleting container app {x}...` message if the managed environment has children - so keeping it does let the user know what the current step is actually doing and lets the user know that the process has progressed.  That said, it is a bit long with it, so I'm happy to remove it... I personally don't have very strong opinions between these two descriptions